doc/governance.rst

```diff
@@ -125,34 +137,39 @@ and the `issue tracker <https://github.com/scikit-learn/scikit-learn/issues>`_.
-At any point during the discussion, any core-developer can call for a vote, which will
-conclude one month from the call for the vote. Any vote must be backed by a
-:ref:`SLEP <slep>`. If no option can gather two thirds of the votes cast, the
-decision is escalated to the TC, which in turn will use consensus seeking with
-the fallback option of a simple majority vote if no consensus can be found
-within a month. This is what we hereafter may refer to as "**the decision making
-process**".
-
-Decisions (in addition to adding core developers and TC membership as above)
+tries to find a resolution that has no open objections among core contributors.
+At any point during the discussion, any core contributor can call for a vote,
+which will conclude one month from the call for the vote. Most votes have to be
+backed by a :ref:`SLEP <slep>`. If no option can gather two thirds of the votes
+cast, the decision is escalated to the TC, which in turn will use consensus
+seeking with the fallback option of a simple majority vote if no consensus can
+be found within a month. This is what we hereafter may refer to as "**the
+decision making process**".
+
+Decisions (in addition to adding core contributors and TC membership as above)
 are made according to the following rules:
 
-* **Minor Documentation changes**, such as typo fixes, or addition / correction of a
-  sentence, but no change of the scikit-learn.org landing page or the “about”
-  page: Requires +1 by a core developer, no -1 by a core developer (lazy
-  consensus), happens on the issue or pull request page. Core developers are
-  expected to give “reasonable time” to others to give their opinion on the pull
-  request if they're not confident others would agree.
+* **Minor Documentation changes**, such as typo fixes, or addition / correction
+  of a sentence, but no change of the ``scikit-learn.org`` landing page or the
+  “about” page: Requires +1 by a maintainer, no -1 by a maintainer (lazy
+  consensus), happens on the issue or pull request page. Maintainers are
+  expected to give “reasonable time” to others to give their opinion on the
+  pull request if they're not confident others would agree.
 
 * **Code changes and major documentation changes**
-  require +1 by two core developers, no -1 by a core developer (lazy
+  require +1 by two maintainers, no -1 by a maintainer (lazy
   consensus), happens on the issue of pull-request page.
 
 * **Changes to the API principles and changes to dependencies or supported
-  versions** happen via a :ref:`slep` and follows the decision-making process outlined above.
+  versions** happen via a :ref:`slep` and follows the decision-making process
+  outlined above.
+
+* **Changes to the governance model** use the same decision process outlined
+  above, but requires no SLEP. However, these changes must be announced on the
+  public mailing list at the beginning of the voting period.
```

Comment:
```suggestion
* **Changes to the governance model** use the [method set out in SLEP020](https://scikit-learn-enhancement-proposals.readthedocs.io/en/latest/slep020/proposal.html).
```
SLEP020 seems to be the most up to date source of how to change the governance model.

Reply:
The wording I have is implementing what we agreed on slep20, I think. Don't you think? The only place where we reference a slep here is when we talk about sleps at the end of this document. I'd be happy to keep sleps out of the rest of the governance doc.

Reply:
The current text didn't seem to cover the "change by PR without a SLEP" option. And instead of adding more text I thought it would be easier to link out.  For me it is Ok to refer to accepted SLEPs to describe the details. But no strong opinion either way regarding creating precedent for doing so.

Reply:
I am not against adding a **reference** to the SLEP in the same way that in the Python documentation, the full specification is usually a hyperlink to the associated PEP.